deploy/csi-azurefile-node.yaml

```diff
@@ -20,7 +20,10 @@ spec:
         kubernetes.io/os: linux
       priorityClassName: system-node-critical
       tolerations:
-        - operator: "Exists"
+        - effect: NoSchedule
+          key: type
+          operator: Equal
```

Comment:
@monaka thanks for the fix. This fix actually means daemonset pod would be scheduled to `virtual-kubelet` node per https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#concepts

Reply:
simply removes `operator: "Exists"` could work, while this driver daemonset would not run on one agent node with taint. This requirement for this PR is actually this:
 - ignore all taints except `virtual-kubelet`
Looks like there is this is not supported in k8s taint functionality.

Reply:
question: is there any side effect if this daemonset pod is scheduled to virtual-kubelet? `operator: "Exists"` will create a new ACI instance which would make user pay more cost?

Reply:
`effect: NoSchedule` means to avoid deploying pods to nodes tagged as `type=virtual-kubelect`.

Reply:
that's opposite, check 
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#concepts
```
A toleration "matches" a taint if the keys are the same and the effects are the same, and
```